dockerfiles/Dockerfile

```diff
@@ -33,10 +33,10 @@ ENV GIT_SECRETS_VERSION=1.3.0
 ENV JSONNET_VENDOR_DIR=/opt/jsonnet-bundler/vendor
 
 RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
-    curl -L https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz | tar -xvz && \
-    pushd openshift-origin-client* && \
+    curl https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz -o oc.tar.gz &&\
```

Comment:
We should probably pin-point a version from https://mirror.openshift.com/pub/openshift-v4/clients/ocp/ so we always know what we are running, otherwise any compatibility break from latest will blow in the hands of someone doing unrelated changes.

Reply:
pinned.